Gemfile

```diff
@@ -37,12 +37,16 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'apartment', '~> 1.2'
+gem "slim-rails"
```

Comment:
I'm not convinced if we should use `slim`/`haml` instead of plain `html.erb`... Maybe we should discuss it in a whole dev group. Personally I prefer working with bare markup, but I will adjust myself to democratic decision :)